### ISSUE TITLE
Don’t allow the user to scroll while the menu is visible

### ISSUE THREAD

**Issue by JeremyGenius** [JeremyGenius/ate#17]
Everything behind the menu should not be interactive while the menu is visible. The text field should be non-scrollable as well.
